scripts/templates/ldrddi.cpp.mako

```diff
@@ -82,7 +82,13 @@ namespace loader
             uint32_t library_driver_handle_count = 0;
 
             result = drv.dditable.${n}.${th.get_table_name(n, tags, obj)}.${th.make_pfn_name(n, tags, obj)}( &library_driver_handle_count, nullptr );
-            if( ${X}_RESULT_SUCCESS != result ) break;
+            if( ${X}_RESULT_SUCCESS != result ) {
+                // If Get Drivers fails with Uninitialized, then update the driver init status to prevent reporting this driver in the next get call.
+                if (${X}_RESULT_ERROR_UNINITIALIZED == result) {
+                    drv.initStatus = result;
+                }
+                continue;
```

Comment:
I realized that we actually don't want to "break", but continue to go thru the drivers now that the DriverGet is allowed to get a failure.